<a href="https://colab.research.google.com/github/stenoe/ABSys/blob/main/notebooks/Volcanoes.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# Estonia's CO2 emission as Volcano

Volcanoes emit annually about 0.28 - 0.36 Gtons CO2 per year. For the sake of simplicity, let's use the maximum of 0.36 Gtons/yr

Estonia emits according to https://globalcarbonatlas.org/emissions/carbon-emissions/ about 10 Mtons/yr (data for 2021).

In [ ]:
# transform Estonian emissions into Gtons/yr
emEstonia = 10. / 1000.
emEstonia

0.01

Let's assume only the approx. 60 active Volcanoes emit all the carbon dioxide for simplicity we can then estimate what one Volcano would emit.

In [ ]:
oneVolcano = 0.36 / 60.
oneVolcano

0.006

We can therefore relate the Estonian emission to volcanoes by

$\frac{0.01}{0.006} = v$

In [ ]:
0.01/0.006

1.6666666666666667

Ok so Estonia's annual CO2 emission is "worth" ~1.7 added volcanoes.